update FAQ page to include some FAQ candidates from Kaldi mailing lists

qindazhu opened the pull request:
update FAQ page to include some FAQ candidates from Kaldi mailing lists

jtrmal commented:
Thanks for doing this!

danpovey commented:
I compiled this, you can now see it at
http://kaldi-asr.org/doc/faq.html


On Thu, Oct 10, 2019 at 5:03 AM jtrmal <notifications@github.com> wrote:

> Thanks for doing this!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/3646?email_source=notifications&email_token=AAZFLO67VXCB2SHNMCKXZ5TQN4KYZA5CNFSM4I7IC4G2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEA37UDQ#issuecomment-540539406>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAZFLO5AQV35RZB53MVKBV3QN4KYZANCNFSM4I7IC4GQ>
> .
>

qindazhu commented:
@danpovey, I have updated FAQ page again at [here](https://github.com/kaldi-asr/kaldi/pull/3651 ).

Updates include:

1. Added some new FAQ candidates from mailing lists (now until Oct. 12).
2. Added section for logo, version and books recommendation for beginners.
